docs/reference/module-overview.md

```diff
@@ -0,0 +1,284 @@
+  modules: {
+    'default-page': {},
+    'landing-page': {}
+  }
+});
+```
+
+```javascript
+// modules/landing-page/index.js
+modules.export = {
+  extend: 'default-page'
+};
+```
+
+You should not use both `improve` and `extend` in a single module.
+
+### `improve`
+
+Similarly to `extend`, `improve` is used to name another existing module. Instead of _adopting_ the existing module's configuration, using `improve` _changes_ the named module's configuration.
+
+**This is only valid in modules that are built independently and then installed into Apostrophe apps**, either on their own or as a part of a bundle. It is most often used to improve core Apostrophe modules.
+<!-- TODO: link to a definition of a bundle when available. -->
+
+You might include this in a stand-alone module that adds functionality to the `@apostrophecms/image` core module:
+
+```javascript
+// index.js
+modules.export = {
+  improve: '@apostrophecms/image'
+  // Additional functionality ...
+};
+```
+
+You should not use both `improve` and `extend` in a single module.
+
+
+### `options`
+
+An object used to add additional, often optional, settings to a module. There are many options available depending on the module type. See the [module configuration options page](/reference/module-options.md) for more information.
+
+### `fields`
+
+[Doc type](/reference/glossary.md#doc) modules have some fields configured by default, such as the `title` and `slug` fields. The `fields` setting is used for additional field management.
+
+The `fields` object is configured with subsections: `add`, `remove`, and `group`.
+
+#### `add`
+
+An object of fields to add to the doc type's schema. See the [field type reference](/reference/field-types/README.md) for more on field type configuration.
+
+```javascript
+// modules/article/index.js
+modules.export = {
+  fields: {
+    add: {
+      subtitle: {
+        label: 'Subtitle',
+        type: 'string'
+      }
+    }
+  }
+};
+```
+
+#### `remove`
+
+An array of field names from the base class module to remove. Some core doc type fields cannot be removed since they required by core functionality.
+
+```javascript
+// modules/spotlight-article/index.js
+modules.export = {
+  extend: 'article',
+  fields: {
+    remove: [ 'subtitle' ]
+  }
+};
+```
+
+#### `group`
+
+An object of field groups. The groupings are used by editing interface. It _does not apply to widget modules_, which do not have a tabbed interface.
+
+Groups are added as an object with their name as the object key and the following properties:
+- `label`: The visible label (a string) on the interface tab
+- `fields`: An array of field names to include in the group
+
+The `@apostrophecms/doc-type` module groups the default fields in a `basics` group, which you can override. If overriding the `basics` group, you will need to add those core fields to a group in the module. Any fields not added to a group will be placed in an "Ungrouped" section in the editing interface.
```

Comment:
"group, which you can override. If overriding the `basics` group," 

remove ",which you can override". and try "You can override the `basics` group by adding the default fields to a group." 

We say 'the default' fields earlier and then call them core fields. I think default is better, but we should stick with it throughout the explanation. We may want to call out or link to 'default' fields in some way, if practical.